examples/distance.c

```diff
@@ -65,8 +65,8 @@ int main(int argc, char *argv[]) {
         radsToDegs(geoHQ2.lon), h3Distance(h3HQ1, h3HQ2),
         haversineDistance(geoHQ1.lat, geoHQ1.lon, geoHQ2.lat, geoHQ2.lon));
     // Output:
-    // origin: (37.775236, 237.580245)
-    // destination: (37.789991, 237.597879)
+    // origin: (37.775236, -122.419755)
+    // destination: (37.789991, -122.402121)
     // grid distance: 2340
-    // distance in km: 2.256850km
+    // distance in km: 2.256853km
```

Comment:
Curious why this changed - FPE? Why are all of the coords here so different in their fractional digits from the previous versions?

Reply:
The only difference I see here is 0.000003km difference on the distance. Not sure if that's important. The coords look like they are correctly updated to `unconstrained_longitude - 360`.

Reply:
...because apparently I can't do simple math, and wasn't expecting a different fraction with the sign change. Stet.